src/unix/bsd/netbsdlike/netbsd/mod.rs

```diff
@@ -2405,6 +2405,16 @@ extern "C" {
         size: ::size_t,
         set: *mut cpuset_t,
     ) -> ::c_int;
+    pub fn pthread_setschedparam(
```

Comment:
As I implied in your ticket, I suggest to put both these in `src/unix/bsd/netbsd/mod.rs` so both systems inherit it.

Reply:
Initially, I did that, but it lead to some errors. Trying once more.